Detectors/FIT/FDD/base/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(MODULE_NAME "FDDBase")
+
```

Comment:
Why is FDD part of the FIT directory?

Reply:
From the management point of view it will be part of FIT in the same way as T0/V0.

Reply:
Thanks for the info.